dipy/reconst/fwdti.py

```diff
@@ -291,10 +304,17 @@ def wls_iter(
     params = np.dot(invWTS2W_WTS2, log_s)
 
     md = (params[0] + params[2] + params[5]) / 3
+    if isinstance(Diso, str):
+        Diso = 3e-3  # Default value if Diso is a string
```

Comment:
not sure to understand this

Reply:
Removed. I only added due to some issue with Diso, seemingly its not there anymore.